client/my-sites/stats/post-performance/index.jsx

```diff
@@ -192,3 +182,15 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state, ownProps ) => {
+	const { site } = ownProps;
+	const query = { status: 'published' };
```

Comment:
@timmyc mm The reasoning here is that we should use the same object `query` to trigger the query (<QueryPosts />) and to get the loading flag. If the query object change over time, we would have one place to make the changes (ease maintenance). This also allows us to ensure we're getting the loading of "our" query (without wondering how the serializing/matching is done by the selectors).